docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,620 @@
+
+1. Ensure that the "id" of the runtime object has a descriptive name such as "TaskPaneRuntime".
+1. Ensure that the "code.page" property of the runtime object is set to the URL of the page that should open in the task pane, such as "https://localhost:3000/taskpane.html".
+1. Ensure that the "actions.view" of the runtime object has a name that describes the content of the page that you set in the preceding step, such as "homepage" or "dashboard". 
+1. Ensure that the "actions.id" of the runtime object has a descriptive name such as "ShowTaskPane" that indicates what happens when the user selects the add-in command button or menu item.
+1. Set the other properties and subproperties of the runtime object as shown in the following completed example of a runtime object. The "type" and "lifetime" properties are required and in Outlook Add-ins (which is the only host that currently supports the unified manifest) they always have the values shown in this example.
+
+    ```json
+    "runtimes": [
+        {
+            "requirements": {
+                "capabilities": [
+                    {
+                        "name": "Mailbox",
+                        "minVersion": "1.3"
+                    }
+                ]
+            },
+            "id": "TaskPaneRuntime",
+            "type": "general",
+            "code": {
+                "page": "https://localhost:3000/taskpane.html"
+            },
+            "lifetime": "short",
+            "actions": [
+                {
+                    "id": "ShowTaskPane",
+                    "type": "openPage",
+                    "view": "homepage"
+                }
+            ]
+        }        
+    ]
+    ```
+
+### Configure the UI for the task pane command
+
+1. Ensure that the extension object for which you configured a runtime has a "ribbons" array property as a peer to the "runtimes" array. There is typically only one extension object in the "extensions" array.
+1. Ensure that the array has an object with array properties named "contexts" and "tabs", as shown in the following example.
+
+    ```json
+    "ribbons": [
+        {
+            "contexts": [
+                // child objects omitted
+            ],
+            "tabs": [
+                // child objects omitted
+            ]
+        }
+    ]
+    ```
+
+1. Ensure that the "contexts" array has strings that specify the windows or panes in which the UI for the task pane command should appear. For example, "mailRead" means that it will appear in the reading pane or message window when an email message is open, but "mailCompose" means it will appear when a new message or a reply is being composed. The following are the allowable values:
+
+    - mailRead
+    - mailCompose
+    - meetingDetailsOrganizer
+    - meetingDetailsAttendee
```

Comment:
These should be in quotation marks to match how you describe "TabDefault" later.

Reply:
> These should be in quotation marks to match how you describe "TabDefault" later.

good catch